utils/ccloud_library.sh

```diff
@@ -905,9 +905,10 @@ function ccloud::create_ccloud_stack() {
   REPLICATION_FACTOR=${REPLICATION_FACTOR:-3}
   enable_ksqldb=${1:-false}
   EXAMPLE=${EXAMPLE:-ccloud-stack-function}
+  CHECK_CREDIT_CARD="${CHECK_CREDIT_CARD:-true}"
 
   # Check if credit card is on file, which is required for cluster creation
-  if [[ $(ccloud admin payment describe) =~ "not found" ]]; then
+  if $CHECK_CREDIT_CARD && [[  $(ccloud admin payment describe) =~ "not found" ]]; then
```

Comment:
Please add output in the `echo` statement in line 912 to nudge the user.  For example:

Change from:

```
    echo "ERROR: No credit card on file. Add a payment method and try again."
```

To something like:

```
    echo "ERROR: No credit card on file. Add a payment method and try again. (If you are using a cloud provider's Marketplace, see documentation for a workaround: https://docs.confluent.io/platform/current/tutorials/examples/ccloud/docs/ccloud-stack.html#running-with-marketplace)"
```

Reply:
is it better to write this message on two lines, with two `echo` it is more readable on the output of the console
```bash
    echo "ERROR: No credit card on file. Add a payment method and try again."
    echo "If you are using a cloud provider's Marketplace, see documentation for a workaround: https://docs.confluent.io/platform/current/tutorials/examples/ccloud/docs/ccloud-stack.html#running-with-marketplace"
```